openapi-circe/src/main/scala/sttp/apispec/openapi/circe/package.scala

```diff
@@ -1,171 +1,18 @@
 package sttp.apispec.openapi
 
-import io.circe.generic.semiauto._
-import io.circe.parser._
-import io.circe.syntax._
-import io.circe.{Encoder, Json, JsonObject}
-import sttp.apispec.{
-  Discriminator,
-  ExampleMultipleValue,
-  ExampleSingleValue,
-  ExampleValue,
-  ExtensionValue,
-  ExternalDocumentation,
-  OAuthFlow,
-  OAuthFlows,
-  Reference,
-  ReferenceOr,
-  Schema,
-  SchemaType,
-  SecurityScheme,
-  Tag
-}
-
-import scala.collection.immutable.ListMap
+import sttp.apispec.AnySchema
 
-package object circe extends SttpOpenAPICirceEncoders
+package object circe extends SttpOpenAPICirceEncoders with SttpOpenAPICirceDecoders {
```

Comment:
Let's use `SttpOpenAPI3_0CirceEncoders` here i.e. `package object circe extends SttpOpenAPICirceEncoders with SttpOpenAPICirceDecoders` => `package object circe extends SttpOpenAPI3_0CirceEncoders with SttpOpenAPICirceDecoders` as long as the most popular open api ui won't support 3.1, 3.0.3 would be mainly used.

Reply:
agreed.